tool/compute_tag_analytics.py

```diff
@@ -279,7 +285,16 @@ def _usage_and_exit(error_message):
     with open(NOOK_EXPORT, mode="r") as nook_export_file:
         nook_export_data = json.load(nook_export_file)
     conversation_tags = nook_export_data[CONVERSATION_TAGS_COLLECTION_KEY]
-    nook_conversations = nook_export_data[CONVERSATIONS_COLLECTION_KEY]
+
+    # If the data is nook_conversations, use it, otherwise try to read from nook_conversation_shards
```

Comment:
nit: This comment doesn't match the logic below which tests nook_conversation_shards first